schema.yaml

```diff
@@ -162,7 +364,7 @@ variables:
     title: Autonomous DB Administrator Password
     description: The password for the admin user (12 to 30 characters [a-zA-Z0-9@#$!&%]$)
     required: true
-    pattern: "^[a-zA-Z0-9@#$!&%]{12,30}$"
+    pattern: ^[A-Za-z][A-Za-z0-9_#\$]+$ 
```

Comment:
Thanks for adding the requirement about first character being a letter, however the length requirement is removed here. Please restore the length requirement 12-30 characters, which means the patter would now be: `^[A-Za-z][A-Za-z0-9_#\$]{11,29}$`

Reply:
Ok, I will fix it with a new commit. I will add log dash sign `^[a-zA-Z0-9@_#$!&%]{12,30}$`